### PR TITLE
Update perf.yml token used to access perf repo

### PR DESCRIPTION
**Pull request by jlb6740**

### CODE REVIEW

**Review by jlb6740 — COMMENTED** [review]
/bench_x64

**Review by jlb6740 — COMMENTED** [review]
/bench_x64

**Review by jlb6740 — COMMENTED** [review]
/bench_x64

**Review by jlb6740 — COMMENTED** [review]
/bench_x64

**Review by jlb6740 — COMMENTED** [review]
/bench_x64

**Review by jlb6740 — COMMENTED** [review]
/bench_x64

**Review by jlb6740 — COMMENTED** [review]
/bench_x64

**Review by jlb6740 — COMMENTED** [review]
/bench_x64

**Review by jlb6740 — COMMENTED** [review]
/bench_x64

**Review by jlb6740 — COMMENTED** [review]
/bench_x64

**Review by jlb6740 — COMMENTED** [review]
/bench_x64

**Review by jlb6740 — COMMENTED** [review]
/bench_x64

**Review by jlb6740 — COMMENTED** [review]
/bench_x64

**Review by jlb6740 — COMMENTED** [review]
/bench_x64

File: .github/workflows/performance.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ jobs:
       - run: echo "$GITHUB_CONTEXT"
       - run: |
           # Create and Push Branch
-          git clone https://jlb6740:${{env.TOKEN}}@github.com/bytecodealliance/wasmtime-sightglass-benchmarking.git
+          git clone https://wasmtime-publish:${{secrets.PERSONAL_ACCESS_TOKEN}}@github.com/bytecodealliance/wasmtime-sightglass-benchmarking.git
```
Comment (alexcrichton) [inline .github/workflows/performance.yml:42]: If this is referencing `secrets` (which I agree is what should be done here) I think the `TOKEN` above is no longer referenced and can be deleted?

**Pull request merged**